app.js

```diff
@@ -0,0 +1,53 @@
+var fecha = new Date(); // objeto fecha actual // var porque es la unica que es modificable dentro de una function
+
+ var numDiasSemana = fecha.getDay();
+ console.log(numDiasSemana);
+function textodia(fecha,numDiasSemana){
+    
+    var diasDeSemana = new Array('Domingo','Lunes','Martes','Miércoles','Jueves','Viernes','Sábado');
+    var diasLetras =  diasDeSemana[fecha.getDay()]; // llamo a fecha
+    
+      
+    
+    return diasLetras;
+ } 
+ console.log(textodia(fecha));
+ let darleColor = document.getElementById('diaSemana');
+     darleColor.style.color = 'rgb(13, 23, 173)'; 
+ if(numDiasSemana % 2 == 0){    // le di color cuando es un dia par
+   let colorDinamico = document.getElementById('diaSemana');
+    console.log('hola esta mostrado en consola');
+    colorDinamico.style.color = '#f37121';
+   }
+ 
+ 
+
+    function textoMes(fecha){
+    var Mes = new Array( 'Enero','Febrero','Marzo','Abril','Mayo','Junio','Julio','Agosto','Septiembre','Octubre','Noviembre','Diciembre');
+    var mesLetra = Mes[fecha.getMonth()];
+    
+    return mesLetra;
```

Comment:
Ya se que ahora estás practicando, pero está bueno siempre formatear el código para que sea más entendible. Creo que en Visual Studio si apretás Shift + Alt + F te lo hace automáticamente. Así queda todo más ordenado y se ve bien dónde empiezan y dónde terminan las funciones, los controles (if, for, etc)

Reply:
Hola Dani, oka. Agregue formato con las reglas de prettier- Code formatter...fui a settings y luego di tilde a format on save para que me formate automaticamente cuando guardo

Reply:
podia hacerlo de una manera por proyecto, me parecio lo mas comoda